api4jenkins/job.py

```diff
@@ -151,7 +151,12 @@ def build(self, **params):
             entry = 'build'
         else:
             entry = 'buildWithParameters'
-        resp = self.handle_req('POST', entry, params=params)
+        files = {}
+        for k in list(params):
+            v = params[k]
+            if hasattr(v, "seek") or hasattr(v, "read"):
```

Comment:
why check `seek` ?

Reply:
Here in this [post](https://stackoverflow.com/questions/1661262/check-if-object-is-file-like-in-python), I have seen one of the comment suggesting to use `seek` for some file systems. We can remove it if you think this is not valid.

Reply:
Yes, i suppose we can remove that. since requests only check attribute `read`, see https://github.com/psf/requests/blob/main/requests/models.py#L190